content/03.results.md

```diff
@@ -162,6 +159,8 @@ The number of SV and CNV breakpoints were significantly correlated across tumors
 Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}E**). 
 We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.
 
+![**Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile. D, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). E, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3.](images/main/Figure3.png){#fig:Fig3}
```

Comment:
We should note that only cancer groups with >=3 samples are shown.

Reply:
I see that noted here:
```
E, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3.](images/main/Figure3.png){#fig:Fig3}
 ```
Did you make the change or am I missing something?